src/advanced/blockchain-explorer.yaml

```diff
@@ -0,0 +1,558 @@
+          $ref: "#/components/schemas/TransactionStatus"
+    ListProof:
+      description: >
+        [Merkle proof](https://exonum.com/doc/advanced/merkelized-list/#merkle-tree-proofs)
+        tying transaction to the tx_hash of the containing block.
+      anyOf:
+        - type: object
+          properties:
+            left:
+              $ref: "#/components/schemas/ListProof"
+            right:
+              $ref: "#/components/schemas/ListProof"
+        - type: object
+          properties:
+            left:
+              $ref: "#/components/schemas/ListProof"
+            right:
+              $ref: "#/components/schemas/Hash"
+        - type: object
+          properties:
+            left:
+              $ref: "#/components/schemas/Hash"
+            right:
+              $ref: "#/components/schemas/ListProof"
+        - type: object
+          properties:
+            left:
+              $ref: "#/components/schemas/ListProof"
+        - type: object
+          properties:
+            val: {}
+    ValidatorId:
+      type: integer
+      minimum: 0
+      maximum: 65535
+      description: Validators identifier.
+    MessageHeader:
+      type: object
+      properties:
+        protocol_version:
+          type: integer
+          minimum: 0
+          maximum: 255
+          description: The major version of the Exonum serialization protocol. Currently, 0.
+        service_id:
+          type: integer
+          minimum: 0
+          maximum: 65535
+          description: Unique service identifier.
+        message_id:
+          type: integer
+          minimum: 0
+          maximum: 65535
+          description: ID of the transaction message.
+        body:
+          type: object
+          description: Body of the transaction.
+        signature:
+          allOf:
+            - $ref: "#/components/schemas/Signature"
+            - description: Signature of the transaction message.
+    SerializedTransaction:
+      allOf:
+        - $ref: "#/components/schemas/MessageHeader"
+        - description: Transaction with the specified hash.
+    BlockHeader:
+      type: object
+      description: Description of a block in blockchain.
+      properties:
+        height:
+          $ref: "#/components/schemas/Height"
+        prev_hash:
+          allOf:
+            - $ref: "#/components/schemas/Hash"
+            - description: Hash of the previous block.
+        proposer_id:
+          allOf:
+            - $ref: "#/components/schemas/ValidatorId"
+            - description: ID of the validator that created an approved block proposal.
+        state_hash:
+          allOf:
+            - $ref: "#/components/schemas/Hash"
+            - description: >
+                Hash of the current
+                [Exonum state](https://exonum.com/doc/glossary/#blockchain-state)
+                after applying transactions in the block.
+        tx_count:
+          type: integer
+          minimum: 0
+          maximum: 4294967295
```

Comment:
I don't think it makes sense to include `u32` or `u64` upper restrictions, tbh, especially for returned types. Additionally, `u64::max_value()` isn't even represented properly in the Swagger doc viewer.